StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsAuthSession.swift

```diff
@@ -77,6 +80,15 @@ struct FinancialConnectionsAuthSession: Decodable {
     var isOauthNonOptional: Bool {
         return isOauth ?? false
     }
+
+    var requiresNativeRedirect: Bool {
+        guard flow == .mxOauthAppToApp else { return false }
+        return url?.hasPrefix("stripe-auth://native-redirect/") ?? false
```

Comment:
now that there's two places where some app-to-app code exists, maybe its worth considering whether any of it is worth reusing (ex. one place changes, do people realize that it matters in multiple places?): 
- https://github.com/stripe/stripe-ios/blob/f276c682/StripeFinancialConnections/StripeFinancialConnections/Source/Web/AuthenticationSessionManager.swift#L149